src/App.js

```diff
@@ -3,13 +3,15 @@ import "./styles.css";
 import { Modal } from "./modal/modal";
 import {days as dayKeys, fullDays} from './helper.js';
 import TaskCard from "./task-card.component";
-import useGetTasks from "./use-get-tasks";
+import {useGetTasks, useGetTaskDates} from './resource-hooks';
 
 export default function App() {
   const [modalVisible, setModal] = useState();
   const [tasks, setTasks] = useState([]);
+  const [taskDates, setTaskDates] = useState([])
```

Comment:
![image](https://user-images.githubusercontent.com/9112811/89368963-08d97c80-d69a-11ea-8109-9a89b0afb9a1.png)
what state now looks like

Reply:
Nice to see the changes we made in state!